javascript/javascript1/week4/voice_assistant.js

```diff
@@ -0,0 +1,94 @@
+const assistant = {};
+const addToDosList = (assistant.toDos = []);
+
+function getReply(command) {
+
+const getUserName = command.split(" ").pop();
+
+  if (command.includes("What is my name?") && Object.keys(assistant).length === 1) {
```

Comment:
Consider reviewing the second condition, since it becomes false if the assistant wants to include any other property other than the ToDos.
You could consider, for example:

Add Line 3: const userName = (assistant.userName = "")
Modify Line 8: if (command.includes("What is my name?") && Object.keys(assistant.userName).length === 0) {

This is just a suggestion, your overall solution looks really good.